signalfx-org-metrics/metrics.yaml

```diff
@@ -1457,6 +1457,164 @@ sf.org.limit.metricTimeSeriesCreatedPerMinute:
   metric_type: gauge
   title: sf.org.limit.metricTimeSeriesCreatedPerMinute
 
-
+sf.org.log.numMessagesReceived:
+  brief: The total number of log messages Splunk Log Observer accepts after filtering and throttling.
+  description: |
+   The total number of log messages Splunk Log Observer accepts after filtering and throttling.
+ 
+       * Dimension(s): `orgId`
+       * Data resolution: 10 second
+  metric_type: counter
+  title: sf.org.log.numMessagesReceived
+ 
+sf.org.log.numContentBytesReceived:
+  brief: The volume of bytes Splunk Log Observer receives from ingesting logs off the wire after filtering and throttling. This content can be compressed.
+  description: |
+   The volume of bytes Splunk Log Observer receives from ingesting logs off the wire after filtering and throttling. This content can be compressed.
+ 
+       * Dimension(s): `orgId`
+       * Data resolution: 10 second
+  metric_type: counter
+  title: sf.org.log.numContentBytesReceived
+ 
+sf.org.log.grossMessagesReceived:
+  brief: The total number of log messages Splunk Log Observer receives before filtering and throttling.
+  description: |
+   The total number of log messages Splunk Log Observer receives before filtering and throttling.
+ 
+       * Dimension(s): `orgId`
+       * Data resolution: 10 second
+  metric_type: counter
+  title: sf.org.log.grossMessagesReceived
+ 
+sf.org.log.numMessageBytesReceived:
+  brief: The number of bytes Splunk Log Observer receives from ingested logs after decompression, filtering, and throttling are complete.
+  description: |
+   The number of bytes Splunk Log Observer receives from ingested logs after decompression, filtering, and throttling are complete.
+ 
+       * Dimension(s): `orgId`
+       * Data resolution: 10 second
+  metric_type: counter
+  title: sf.org.log.numMessageBytesReceived
+ 
+sf.org.log.grossContentBytesReceived:
+  brief: The volume of bytes Splunk Log Observer receives from ingesting logs off the wire before filtering and throttling. This content can be compressed.
+  description: |
+   The volume of bytes Splunk Log Observer receives from ingesting logs off the wire before filtering and throttling. This content can be compressed.
+ 
+       * Dimension(s): `orgId`
+       * Data resolution: 10 second
+  metric_type: counter
+  title: sf.org.log.grossContentBytesReceived
+ 
+sf.org.log.numMessagesReceivedByToken:
+  brief: The total number of log messages Splunk Log Observer accepts for a specific access token after filtering and throttling.
+  description: |
+   The total number of log messages Splunk Log Observer accepts for a specific access token after filtering and throttling.
+ 
+       * Dimension(s): `orgId`
+       * Data resolution: 10 second
+  metric_type: counter
+  title: sf.org.log.numMessagesReceivedByToken
+ 
+sf.org.log.numMessagesDroppedOversize:
+  brief: The number of log messages Splunk Log Observer receives that are too large to process.
```

Comment:
It could be helpful to put the maximum size here so that an additional search isn't needed to find it.

Reply:
this maximum size depends on the kafka broker configs and hence it's specific to a cluster. Hence having that information here will not be any value as it can change if we change the cluster nodes to different type